tailwind.config.ts

```diff
@@ -6,6 +6,7 @@ const config: Config = {
     "./src/pages/**/*.{js,ts,jsx,tsx,mdx}",
     "./src/components/**/*.{js,ts,jsx,tsx,mdx}",
     "./src/app/**/*.{js,ts,jsx,tsx,mdx}",
+    "./src/common/*.{js,ts,jsx,tsx,mdx}",
```

Comment:
@Mubashar1009 why are you using this ,  we are not using `js` or `jsx`

Reply:
actually it is not using now so i removed it 
it is a good for defining these files extension also according to the documentation perhaps in future some time of this extension file can be need

Reply:
okay nice